tests/test_apiclient.py

```diff
@@ -12,7 +13,7 @@ def test_constructor_with_success(self):
         client = RevAiAPIClient(token)
 
         headers = client.session.headers
-        assert headers.get('User-Agent') == 'python_sdk'
+        assert headers.get('User-Agent') == 'python_sdk-{}'.format(__version__)
```

Comment:
I think this test should use string literals.
```suggestion
        # Setup
        __version__ = 'test-version'
        
        # Assert
        assert headers.get('User-Agent') == 'python_sdk-test-version'
```

Reply:
Currently, the test just repeats the implementation code.

Reply:
@amikofalvy do you mean we should check that the user-agent string is `'python_sdk-{}'.format(__version__)` versus `python_sdk-test-version'`? I personally would have put it as Kyle did it i think but i don't have strong opinion on this

Reply:
Talked with @amikofalvy. We ended up agreeing on this implementation